h/groups/models.py

```diff
@@ -109,12 +109,14 @@ def created_by(cls, session, user):
 
 USER_GROUP_TABLE = sa.Table(
     'user_group', Base.metadata,
+    sa.Column('id', sa.Integer, primary_key=True),
```

Comment:
For consistency with the rest of our models this should also specify `autoincrement=True`. Strictly speaking I don't believe it's necessary since sqla 1.1 which introduced `autoincrement='auto'` as the default, but no other models in our code define an integer PK without `autoincrement=True` so it took me a minute to work out whether it was correct.